### PR TITLE
Add service and param commands

### PR DESCRIPTION
**Pull request by dbruggner**

### CODE REVIEW

File: ros2/param.bzl [hunk 1]
```diff
@@ -0,0 +1,25 @@
+""" Implements a macro for setting up a ROS2 param app.
```
Comment (mvukov) [inline ros2/param.bzl:1]: What's the motivation for ros2_param macro? I though ros2_param binary (defined in https://github.com/mvukov/rules_ros2/blob/main/ros2/BUILD.bazel#L32) is sufficient. BTW, you can alias it in your workspace as https://github.com/mvukov/rules_ros2/blob/main/examples/BUILD.bazel#L2.

File: ros2/ros2_service.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Milan Vukov
```
Comment (mvukov) [inline ros2/ros2_service.py:1]: It's your work, just put your name here. And update year to 2023 ;).

**Review by mvukov — COMMENTED** [review]
Nice work :)

**Comment by mvukov** [comment]
BTW, please update the main readme. Please add ros2_service in the list of utils.

File: ros2/param.bzl (hunk 1 above)
Comment (dbruggner) [reply ros2/param.bzl]: you're right!

**Comment by dbruggner** [comment]
Done

**Review by mvukov — APPROVED** [review]
:rocket:

**Pull request merged**